rc_simulator/building_physics.py

```diff
@@ -257,6 +257,9 @@ def solve_building_energy(self, internal_gains, solar_gains, t_out, t_m_prev):
             # --> rc_model_function_1(...)
             self.energy_demand = 0
 
+            from numpy import nan
```

Comment:
Could you please bring the import command to the top of the script. So in line 84. This way all external modules can be known and referenced in the start